### ISSUE TITLE
bin/ewallet config command failing on master

### ISSUE THREAD

**Issue by sirn** [omisego/ewallet#972]
```
18:07:39.755 [error] GenServer EWalletConfig.Config terminating
** (KeyError) key EWallet.ReleaseTasks.CLIUser not found in: %{ActivityLogger.System => %{identifier: nil, type: "system"}, EWalletConfig.StoredSetting => %{identifier: :id, type: "setting"}}
    (stdlib) :maps.get(EWallet.ReleaseTasks.CLIUser, %{ActivityLogger.System => %{identifier: nil, type: "system"}, EWalletConfig.StoredSetting => %{identifier: :id, type: "setting"}})
    (activity_logger) lib/activity_logger/activity_log.ex:89: ActivityLogger.ActivityLog.get_type/1
    (activity_logger) lib/activity_logger/activity_log.ex:163: ActivityLogger.ActivityLog.build_attrs/3
    (activity_logger) lib/activity_logger/activity_log.ex:157: ActivityLogger.ActivityLog.insert/3
    (ewallet_config) lib/ewallet_config/repo.ex:20: EWalletConfig.Repo.handle_perform_result/3
    (ewallet_config) lib/ewallet_config/setting.ex:198: EWalletConfig.Setting.update/2
    (ewallet_config) lib/ewallet_config/setting.ex:219: anonymous fn/2 in EWalletConfig.Setting.update_all/1
    (elixir) lib/enum.ex:1331: anonymous fn/3 in Enum.map/2
```

**Comment by sirn** [comment]
@T-Dnzt can you take a look? Not sure why this regressed

**Comment by T-Dnzt** [comment]
It seems it doesn't go through `apps/ewallet/lib/ewallet/application.ex` anymore. That's where the registration of a new Activity Logger originator happens:

```
    ActivityLogger.configure(%{
      EWallet.ReleaseTasks.CLIUser => %{type: "cli_user", identifier: nil}
    })
```

I'm not sure why tho. Does the `ewallet` subapp gets loaded properly and before we try to change the settings? 🤔

**Comment by sirn** [comment]
Turns out this was due to E2E not setting the `KEYCHAIN_SECRET_KEY` environment in master. Oops. Sorry for the noise. This should be fixed in #971
